src/resources/js/views/manager.js

```diff
@@ -629,6 +648,7 @@ tribe.events.views.manager = {};
 	 */
 	obj.ready = function() {
 		obj.selectContainers();
+		obj.normalizeData();
```

Comment:
Let's move this into `obj.setup()`, just after `var $container`, and pass in the `$container` into the function. We have the potential case where we can have more than 1 container printed out on a page, we should normalize for each container.